tests/integrations/tests/certificate_settlement/node_state.rs

```diff
@@ -0,0 +1,59 @@
+use std::time::Duration;
```

Comment:
nit: this file which contains only 1 test to get network status is in a folder named `certificate_settlement`

Reply:
I had dilemma where to put it. Should I make a separate folder for this particular test?